src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
+    <Version>0.8.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Personalizer</AssemblyName>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Personalizer API;REST HTTP client</PackageTags>
+    <PackageReleaseNotes>
+    <![CDATA[
+    Microsoft Cognitive Services Personalizer SDK.
+
+    This package calls the API version "v0.8.0-preview"
```

Comment:
I am not sure that is correct. Please check what REST API version this corresponds to

Reply:
fixed